index.js

```diff
@@ -0,0 +1,17 @@
+// Написать функцию persistence, которая принимает положительный параметр num и возвращает число, которое нужно умножить цифры в num, пока не дойдете до единственной цифры.
+
+//O(kn) - the coefficient depends on the number
+function persistence(number) {
+  let [count, result] = [0, number];
```

Comment:
too strange structure, better to create 2 separate variables

Reply:
refactored it